### ISSUE TITLE
The performance of the model I trained is not good.

### ISSUE THREAD

**Issue by bluu0125** [hoiliu-0801/DQ-DETR#3]
Hello, I trained a model on 4 RTX 4090 GPUs using the VisDrone dataset, but only achieved AP=34.0, AP50=55.7, and AP75=34.8, which is about 3-5 points lower than the results in your paper. Could you advise on what aspects I should pay attention to during training?Thank you.

**Comment by hoiliu-0801** [comment]
1. What evaluation metric did you use?
2. What settings did you apply? Check the batch size, epochs, etc., to optimize the performance. I will release the pre-trained weights for the Visdrone dataset and report those hyperparameters I used in the future.

**Comment by bluu0125** [comment]
> 1. What evaluation metric did you use? You can use the official evaluation tools for better performance: https://github.com/VisDrone/VisDrone2018-DET-toolkit.

Thank you for your reply.  I was previously using the existing COCO evaluation tools in the code, and now I’m trying to use the official evaluation tools. However, I noticed that the results are not as good. Upon inspection, I found that many of the predicted bounding boxes in the validation set are correct, but the labels are wrong. However, the results obtained from the original COCO evaluation tools didn’t show most labels as incorrect. My operation was to add "--save_results" in DQ_eval.sh to save the validation set predictions and then manually convert the results into the format required by the official evaluation tools. Could you please let me know if there's something wrong with my approach?

**Comment by hoiliu-0801** [comment]
Yes, you're right. However, I don't recall if there was such a significant issue with the COCO evaluation. Thank you for sharing. You can compare the results with the official evaluation for verification and update the findings here. That would be helpful.

**Comment by bluu0125** [comment]
@hoiliu-0801 I have solved my problem by changing ”_res_bbox = outbbox“ to ”_res_bbox = res['boxes']“ when saving the results. Thank you for your help.
